### DISCUSSION TITLE
grouping of replicates by treatment

### DISCUSSION THREAD

**Discussion by sylestiel** [GreenleafLab/ArchR#1769] [General]
Hi,

 I have 2 treatment conditions and 3 replicates per condition. How can I group the replicates by treatment condition so as to compare the two treatments? This is with respect to 10X multiome datasets.

Thanks!

**Comment by rcorces** [comment]
I'm not sure I understand the question. cells are grouped based off of values in `cellColData` you can add whatever new columns you want and define cell groupings however you want

**Reply by sylestiel** [reply]
I tried to use the accessor function but was not successful at generating the groups of interest.

proj$condition <- ("A", "B")
proj$A <- ("A1", "A2", "A3")
proj$B <- ("B1", "B2", "B3")

Can you tell me what I'm doing wrong?

Thanks!

**Reply by rcorces** [reply]
https://www.archrproject.com/bookdown/manipulating-an-archrproject.html
